### ISSUE TITLE
Intermittent Service

### ISSUE THREAD

**Issue by eskivelevolution** [Microsoft/BotFramework-WebChat#1288]
Hi,

We received this error 'ERR_SSL_VERSION_INTERFERENCE'
and 'ERR_CONNECTION_RESET' when my chat is trying to connect.

Sometimes the direct line get the connection..

Direct Line has problems?

Thank you.
![image](https://user-images.githubusercontent.com/19326711/47881729-c1a7c300-de05-11e8-9925-129617f9cae6.png)

**Comment by vincec-msft** [comment]
Yes, sorry for the pain. We are seeing a very high volume of traffic to our East US datacenters and are working on a hotfix to deploy tonight.

See also https://ms.portal.azure.com/#blade/Microsoft_Azure_Health/AzureHealthBrowseBlade/serviceIssues

**Comment by vincec-msft** [comment]
A hotfix has been deployed and Direct Line is looking healthy again.

https://portal.azure.com/#blade/Microsoft_Azure_Health/AzureHealthBrowseBlade/serviceIssues hasn't been updated yet. We're still watching to make sure the system is happy.
